demo/__init__.py

```diff
@@ -12,6 +12,11 @@
 
 from six.moves import range
 
+
+local = 'localhost'
+remote = 'repo.uptane.org'
```

Comment:
They're there to be plugged in optionally in the fields below. (e.g. `DIRECTOR_SERVER_HOST = local` can be swithced to `DIRECTOR_SERVER_HOST = remote`.